torch/distributed/_tensor/_collective_utils.py

```diff
@@ -158,3 +160,128 @@ def mesh_all_to_all(
             async_op=async_op,
         )
     return work
+
+
+def spec_to_bytes(spec: "placement_types.DTensorSpec") -> int:
```

Comment:
To clarify, the result is in  bytes right?

Reply:
yep it's #num bytes